compiler/src/dotty/tools/dotc/tastyreflect/KernelImpl.scala

```diff
@@ -282,6 +282,9 @@ class KernelImpl(val rootContext: core.Contexts.Context, val rootPosition: util.
     if (self.symbol.signature == core.Signature.NotAMethod) None
     else Some(self.symbol.signature)
 
+  def Select_apply(qualifier: Term, symbol: Symbol)(implicit ctx: Context): Select =
+    withDefaultPos(implicit ctx => tpd.Select(qualifier, Types.TermRef(qualifier.tpe, symbol)))
```

Comment:
This implicit function type will break the scala 2.12 build

Reply:
Which is only tested after the merge

Reply:
We already have it in the code above?

Reply:
Then it fine